app/src/androidTest/java/net/skyscanner/backpack/snackbar/BpkSnackbarTests.kt

```diff
@@ -135,19 +141,38 @@ class BpkSnackbarTests : BpkSnapshotTest() {
 
   private inline fun capture(crossinline what: AsyncSnapshot.() -> BpkSnackbar) {
     val asyncScreenshot = prepareForAsyncTest()
+    var snackbar: BpkSnackbar? = null
```

Comment:
this is not ideal, but the snackbar needs to be shown/setup on the ui thread, espresso needs to be not run on the ui thread